code/datums/mood_events/generic_negative_events.dm

```diff
@@ -403,6 +403,13 @@
 	mood_change = -1
 	timeout = 30 SECONDS
 
+//MONKESTATION EDIT START
+/datum/mood_event/watersprayed/cat
```

Comment:
could just move this to `monkestation/code/modules/datums/mood_events/generic_negative_events.dm`